doc/changelog.rst

```diff
@@ -4,8 +4,13 @@ Changelog
 Changes in Version 4.3.3
 ------------------------
 
-Version 4.3.3 documents support for :ref:`CSFLE on-demand credentials` for cloud KMS providers, and fixes the following bugs:
+Version 4.3.3 documents support for the following:
 
+- :ref:`CSFLE on-demand credentials` for cloud KMS providers.
+- Authentication support for :ref:`EKS Clusters`.
```

Comment:
Can you move the CSOT docs line here as well?

Reply:
Done